docs/source/release_notes/release-notes.rst

```diff
@@ -40,8 +40,12 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |fixed|
+         - Timelock server now supports HTTP/2 and the AtlasDB HTTP clients enable a required GCM cipher suite.
+           This feature improves performance of the Timelock server. Any client that wishes to connect to the timelock server
+           via HTTP2 must add the jetty_alpn_agent as a javaAgent JVM argument, otherwise connections will fall back to
+           HTTP/1.1 and performance may be slow.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1594>`__)
```

Comment:
Nicely done! A few suggestions:

- I think this should be `new`; not supporting HTTP/2 doesn't really mean Timelock was broken at all...
- nit: We should standardise `HTTP/2` (line 46)

Also, it would be nice if we could give a concrete example of how to configure the Jetty ALPN agent (i.e. `-javaagent:service/lib/jetty-alpn-agent-2.0.6.jar`). This could be another (small) PR and in a separate section of the docs (and we can include a link here on how to do it).

@rhero can advise better here.